app/models.py

```diff
@@ -5,3 +5,149 @@
+    registration_date = db.Column(db.DateTime)  # was `regdate`
+    last_login_date = db.Column(db.DateTime)  # was `lastdate`
+    last_login_ip = db.Column(db.DateTime)  # was `lastip`
+    star = db.Column(db.Integer)
+    # score temporarily not included
+    post_num = db.Column(db.Integer, unsigned=True)  # was `post`
+    reply_num = db.Column(db.Integer, unsigned=True)  # was `reply`
+    water_num = db.Column(db.Integer, unsigned=True)  # was `water`
+    sign_num = db.Column(db.Integer, unsigned=True)  # was `sign`
+    # rights temporarily not included
+    # newmsy temporarily not included
+    # extr temporarily not included
+    last_post_time = db.Column(db.DateTime)  # was `lastpost`
+    current_board = db.Column(db.Integer)  # was `nowboard`
+    user_agent = db.Column(db.String(128))  # was `onlinetype`
+    # logininfo temporarily not included
+    # code temporarily not included
+    # other2 3 4 5 6 temporarily not included
+
+
+class Board(db.Model):
+    __tablename__ = 'boards'
+
+    bid = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(32), unique=True)  # was `bbstitle`
+    invisible = db.Column(db.Boolean, default=False)  # was `hide`
+
+
+class Comment(db.Model):
+    __tablename__ = 'comments'
+
+    cid = db.Column(db.Integer, primary_key=True)  # was `id`
+    pid = db.Column(db.Integer)  # was `fid`
+    uid = db.Column(db.Integer)  # was `author`
+    content = db.Column(db.Text)  # was `text`
+    time = db.Column(db.DateTime)
+    deleted = db.Column(db.Boolean, default=False)  # was `visible`
+
+
+class Message(db.Model):
+    __tablename__ = 'messages'
+
+    mid = db.Column(db.Integer, primary_key=True)
+    sender_uid = db.Column(db.Integer)
+    receiver_uid = db.Column(db.Integer)
+    content = db.Column(db.Text)
+    time = db.Column(db.DateTime)
+    is_read = db.Column(db.Boolean, default=False)
+    # ruser, rmsg, rbid, rtid, rpid deleted
+
+
+class Notification(db.Model):
+    __tablename__ = 'notifications'
+
+    receiver_uid = db.Column(db.Integer)
+    time = db.Column(db.DateTime)
+    ntype = db.Column(db.Integer)
+    # Type of the notification: reply, at or quote
+    pid = db.Column(db.Integer)
+    cid = db.Column(db.Integer)
+
+
+class Post(db.Model):
+    __tablename__ = 'posts'
+
+    pid = db.Column(db.Integer, primary_key=True)  # was `fid`
+    bid = db.Column(db.Integer)
+    tid = db.Column(db.Integer)
+    title = db.Column(db.String(32))
+    author_uid = db.Column(db.Integer)  # was `author`
+    content = db.Column(db.Text)  # was `text`
+    # ishtml temporarily not included
+    # attachs temporarily not included
+    created_at = db.Column(db.DateTime)  # was `replytime`
+    updated_at = db.Column(db.DateTime)  # was `updatetime`
+    signature = db.Column(db.Text)
+    user_agent = db.Column(db.String(128))  # was `type`
+    ip = db.Column(db.String(64))
+    # lzl temporarily not included
+
+
+class Sign(db.Model):
+    __tablename__ = 'sign'
```

Comment:
So this is for login log? Actually I don't think we need it if we just use it for calculate the `num_login`